### DISCUSSION TITLE
How to properly upgrade Tauri in a project?

### DISCUSSION THREAD

**Discussion by hoichi** [tauri-apps/tauri#2119] [Q&A] [answered]
First of all, thank you for your tremendous work. I'm hoping Tauri will go far, and I'm hoping to do great things with its help too :metal:

Now the question. Currently having Tauri in your project means several Cargo packages and several npm packages. I see new versions appearing, and I see that various packages have various beta numbers. So, my question is: once you've started a project, how do you upgrade Tauri. In particular:
- Should `tauri` in Cargo and `@tauri/api` in npm be in sync
- Should various other dependencies, like CLI or `tauri-bild` be in sync (I'm guessing not?).

**Comment by e-l-i-j-a-h** [comment, marked as answer]
![tauri update](https://user-images.githubusercontent.com/13988479/124520203-b88f0100-ddb9-11eb-9679-8f2c48371f33.png)

The way I upgrade Tauri is by running `yarn tauri deps update`. It should update `@tauri-apps/api`, `@tauri-apps/cli`, and everything else.

**Comment by liudonghua123** [comment]
`yarn tauri deps update` seems deprecated or removed, I got `error: The subcommand 'deps' wasn't recognized` errors. I use `yarn upgrade @tauri-apps/cli @tauri-apps/api --latest` to update npm packages, and modifiy `src-tauri/Cargo.toml` manually now, see https://tauri.app/v1/guides/development/updating-dependencies/.
